### PR TITLE
Added a section with derived work in "Short, friendly base32 slugs from timestamps"

### PR DESCRIPTION
**Pull request by lmammino**
I have created a Rust crate that follows your approach for id generation. This PR mentions it at the bottom of the article, just in case you'd like to give a mention to derived work.

In any case, thanks for the very informative article, it's a very clever trick and I love reading about it and trying to re-implementing it in Rust.

### CODE REVIEW

File: content/fragments/base32-slugs.md [hunk 1]
```diff
@@ -46,5 +46,10 @@ var lexicographicBase32Encoding = base32.NewEncoding(lexicographicBase32).
 
 So now when I list atoms from my filesystem or in S3, they come back in the same order that I wrote them.
 
+
+## Derivatives
```
Comment (brandur) [inline content/fragments/base32-slugs.md:50]: I have some custom Markdown rendering that gives headers stable anchors, but they have to be defined manually:
```suggestion
## Derivatives (#derivatives)
```

File: content/fragments/base32-slugs.md [hunk 2]
```diff
@@ -46,5 +46,10 @@ var lexicographicBase32Encoding = base32.NewEncoding(lexicographicBase32).
...
+
+- This post inspired a [Rust crate](https://crates.io/crates/lexicoid)
```
Comment (brandur) [inline content/fragments/base32-slugs.md:52]: I'm a bit of a mizer about punctuation:

```suggestion
- This post inspired a [Rust crate](https://crates.io/crates/lexicoid).
```

**Review by brandur — COMMENTED** [review]
Thanks Luciano! Took a look at the Rust code — great stuff. Thanks.

**Comment by lmammino** [comment]
My pleasure! I am glad you liked it. I just applied your suggestions.

**Comment by brandur** [comment]
Thanks! Merging this, but I'm having a bit of trouble with ImageMagick in my build, so it won't be visible right away.

**Pull request merged**